src/astro/databases/aws/redshift.py

```diff
@@ -0,0 +1,116 @@
+"""AWS Redshift table implementation."""
+from typing import List
+
+import pandas as pd
+import sqlalchemy
+from airflow.providers.amazon.aws.hooks.redshift_sql import RedshiftSQLHook
+from sqlalchemy import create_engine
+from sqlalchemy.engine.base import Engine
+
+from astro.constants import DEFAULT_CHUNK_SIZE, LoadExistStrategy
+from astro.databases.base import BaseDatabase
+from astro.files import File
+from astro.settings import REDSHIFT_SCHEMA
+from astro.sql.table import Metadata, Table
+
+DEFAULT_CONN_ID = RedshiftSQLHook.default_conn_name
+
+
+class RedshiftDatabase(BaseDatabase):
+    """
+    Handle interactions with Redshift databases.
+    """
+
+    DEFAULT_SCHEMA = REDSHIFT_SCHEMA
+
+    illegal_column_name_chars: List[str] = ["."]
```

Comment:
Do we know these chars are not valid col names in reshifts?

Reply:
yes, that is correct, Redshift does not accept `.` column name characters